src/common/components/MarkDownParser/Quote.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Box } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import quoteImg from '../../img/quote.png';
+
+const Quote = ({ children, ...props }) => {
```

Comment:
We should have the 4 versions provided in the design, the version used will be randomly selected

Reply:
You have to create each of the component's version